products/data_warehouse/backend/migrations/0007_backfill_saved_query_origin.py

```diff
@@ -0,0 +1,49 @@
+from django.db import migrations
+
+CHUNK_SIZE = 1000
+
+
+def forwards_func(apps, schema_editor):
+    SavedQuery = apps.get_model("data_warehouse", "DataWarehouseSavedQuery")
+
+    chunk = []
+    for query in (
+        SavedQuery.objects.filter(managed_viewset__isnull=False).only("id", "origin").iterator(chunk_size=CHUNK_SIZE)
```

Comment:
i think you want `managed_viewset__isnull=True` right? you'd be overwriting non-null origins with this.